docs/user/disconnected-cluster.md

```diff
@@ -0,0 +1,187 @@
+# Installing a Flight Control in a Disconnected Cluster Environment
+
+> 📌 Please note, that version `0.4.0-rc3` are used for demonstration purposes only
+
+> 📌 All `sha256` are valid for `0.4.0-rc3` version only
+
+## Log in
+
+### login to hub
+
+```shell
+ssh root@<hub-hostname>
+```
+
+### all operations should be executed from `kni` user
+
+```shell
+su - kni
+```
+
+### login to cluster
+
+```shell
+export KUBECONFIG=clusterconfigs/auth/kubeconfig
+```
+
+## Configure mirroring `icsp`
+
+### check whether `quay.io` mirror is configured in `icsp`
+
+```shell
+oc get imagecontentsourcepolicy image-policy -o yaml
+```
+
+look for
+
+```yaml
+  - mirrors:
+    - <registry-hostname>:5005/flightctl
+    source: quay.io/flightctl
+  - mirrors:
+    - <registry-hostname>:5005/sclorg
+    source: quay.io/sclorg
+```
+
+if not present, add it using
+
+```shell
+oc edit imagecontentsourcepolicy image-policy
+```
+
+## Mirror quay.io images
+
+### see list of images to use
+
+```shell
+helm show values oci://quay.io/flightctl/charts/flightctl --version=0.4.0-rc3
+```
+
+### mirror flightctl images
```

Comment:
Better to have a small script iterating over all images: mirroring and replacing tag in values.yaml

Reply:
I'm not going to write and support a script. As well you saw that there are images that are not mentioned there, like keycloak.

Reply:
So what are the instruction to end user. i.e mirror images, and then change helm defaults.yaml to sha, etc